modules/op-release-notes-1-16.adoc

```diff
@@ -0,0 +1,603 @@
+    ref:
+      resolver: http
+      params:
+      - name: url
+        value: https://raw.githubusercontent.com/openshift-pipelines/tektoncd-catalog/p/stepactions/stepaction-git-clone/0.4.1/stepaction-git-clone.yaml
+    params:
+    - name: URL
+      value: $(params.url)
+    - name: OUTPUT_PATH
+      value: $(workspaces.output.path)
+----
+
+* With this update, the `openshift-pipelines` namespace includes versioned tasks alongside standard tasks. For example, there is a `buildah` standard task and a `buildah-1-16-0` versioned task. While the standard task might be updated in subsequent releases, the versioned task remains exactly the same as it was in a specified version, except for correction of errors.
+
+* With this update, you can configure the `FailurePolicy`, `TimeoutSeconds`, and `SideEffects` options for webhooks for several components of {pipelines-shortname} by using the `TektonConfig` CR.
++
+.Example configuration of webhooks options for the `pipeline` component
+[source,yaml]
+----
+apiVersion: operator.tekton.dev/v1alpha1
+kind: TektonConfig
+metadata:
+  name: config
+spec:
+  pipeline:
+    options:
+      webhookConfigurationOptions:
+        validation.webhook.pipeline.tekton.dev:
+          failurePolicy: Fail
+          timeoutSeconds: 20
+          sideEffects: None
+        webhook.pipeline.tekton.dev:
+          failurePolicy: Fail
+          timeoutSeconds: 20
+          sideEffects: None
+----
++
+You can use similar configuration for webhooks in the `triggers`, `pipelinesAsCode`, and `hub` components.
+
+
+[id="triggers-new-features-1-16_{context}"]
+=== Triggers
+
+* With this update, the `readOnlyRootFilesystem` parameter for the triggers controller, webhook, Core Interceptor, and event listener is set to `true` by default to improve security and avoid being flagged by the security scanner.
+
+* With this update, you can configure {pipelines-shortname} triggers to run event listeners as a non-root user within the container. To set this option, set the parameters in the `TektonConfig` CR as shown in the following example:
++
+.Example of configuring trigger event listeners to run as non-root
+[source,yaml]
+----
+apiVersion: operator.tekton.dev/v1alpha1
+kind: TektonConfig
+metadata:
+  name: config
+spec:
+  trigger:
+    options:
+      disabled: false
+      configMaps:
+        config-defaults-triggers: 
+          data:
+            default-run-as-non-root: "true"
+            default-run-as-user: "65532"
+            default-run-as-group: "65532"
+            default-fs-group: "65532"
+----
++
+Optionally, you can set the values of the `default-run-as-user` and `default-run-as-group` parameters to configure the numeric user ID and group ID for running the event listeners in containers; {pipelines-shortname} sets these values in the pod security context and container security context for event listeners. If you use empty values, the default user ID and group ID of `65532` are used.
```

Comment:
The semicolon means that this sentence is very long. Use a full stop to break up the two sentences.

![Screenshot from 2024-10-04 15-46-44](https://github.com/user-attachments/assets/f2af6fa0-b4d3-4c7c-a9ca-a6113ba73f10)